waiting-for-finish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo "waiting for all background process to stop..."
+declare -a arr=("npm" "java")
```

Comment:
But if I only check node, when the container stops, it will still need to
kill npm itself

2016-06-28 11:06 GMT-05:00 Xingchen Hong <notifications@github.com>:

> In waiting-for-finish.sh
> <https://github.com/Zodiase/map-visualizer/pull/15#discussion_r68788272>:
>
> > @@ -0,0 +1,17 @@
> > +#!/bin/bash
> > +
> > +echo "waiting for all background process to stop..."
> > +declare -a arr=("npm" "java")
>
> Should check for node rather than npm.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Zodiase/map-visualizer/pull/15/files/1c3d4e5ef0ec58890c6b2a5a60b21090902a2c76#r68788272>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJZCbQGjhJC4wtgDgklAzhy4GGwgJfVeks5qQUaZgaJpZM4JAOXy>
> .
>